corehq/motech/repeaters/const.py

```diff
@@ -6,7 +6,10 @@
 
 MAX_RETRY_WAIT = timedelta(days=7)
 MIN_RETRY_WAIT = timedelta(minutes=60)
-RATE_LIMITER_DELAY_RANGE = (timedelta(minutes=0), timedelta(minutes=15))
+RATE_LIMITER_DELAY_RANGE = (
+    timedelta(minutes=settings.MIN_REPEATER_RATE_LIMIT_DELAY),
+    timedelta(minutes=settings.MAX_REPEATER_RATE_LIMIT_DELAY),
+)
```

Comment:
I'm curious how you feel about defaults? When we reference these values directly, they need to be present in our `settings` object or an error is thrown. I've been looking at using syntax like `getattr(settings, 'MAX_REPEATER_RATE_LIMIT_DELAY', 15)` instead, as it avoids the need to put everything in settings.  At a certain point, when we place everything in settings, I'd imagine the settings file becomes pretty overwhelming. Provided we have sensible defaults, using `getattr` reduces the overall settings size at the cost of some discoverability.

I wonder if the divide is between settings the user will know they need to tweak without any knowledge of our system (i.e. custom URLs) vs technical details. Not sure where this should fall.

Reply:
Yeah I like the sound of that better. I would think this is a technical detail that users will likely never need to think about so going with your approach seems objectively better.

Reply:
> the cost of some discoverability.

I suppose if we were really concerned about this, we could create some helper function to do exactly this, but just makes it easy to grep for. Though for now, I'm not too concerned about that.

Reply:
52d49d8cf0a6e516b531a7e696cf9a28dd98e62f